packages/@aws-cdk/aws-eks/test/integ.eks-cluster.expected.json

```diff
@@ -28,6 +28,53 @@
         }
```

Comment:
Please confirm that you have deployed this from scratch and that encryption was enabled as a result.

Reply:
Confirmed the encryption was enabled after running the integration test and checked the KMS CMK was used for encrypt/decrypt in CloudTrail log.

Reply:
I am curious how the integ test actually passed given [this](https://github.com/aws/aws-cdk/issues/9140#issuecomment-670030611)

Reply:
Our observation is the `aws-sdk` version in Lambda runtime has been bumped to `2.712.0` silently for most (if not all) regions. See [this](https://github.com/aws/aws-cdk/issues/9140#issuecomment-671393634)

Reply:
Screenshot taken in AWS Console after running the integ test from scratch a few days ago
<img width="1054" alt="eks-cluster-integ-test-console" src="https://user-images.githubusercontent.com/473354/89797392-77687f80-db5d-11ea-8151-eeaef720d221.png">